.github/workflows/openmdao_test_workflow.yml

```diff
@@ -220,14 +220,17 @@ jobs:
           echo "============================================================="
           echo "Build the docs"
           echo "============================================================="
-          python build_source_docs.py
-          jupyter-book build -W --keep-going openmdao_book
-          python copy_build_artifacts.py
+          ./build_all_docs.sh
```

Comment:
Just to make sure, have you verified that an error raised from the jupyter-book build will trigger an error on this workflow?

Reply:
ah.. no, I was not aware of that wrinkle...  I can back this change out...